src/main/java/service/ReportService.java

```diff
@@ -0,0 +1,6 @@
+package service;
+
+public interface ReportService {
+
```

Comment:
remove the first empty line in all classes/interfaces/methods where it is present
```suggestion
```